app/src/features/FhirResourceTree/FhirResourceTree.tsx

```diff
@@ -100,16 +102,21 @@ const FhirResourceTree = (): JSX.Element => {
         ({ path }) => path === node.path
       );
       if (!selectedNodeAttribute) {
-        const attribute = await createAttribute({
-          attributeRequest: {
-            definition_id: node.type,
-            path: node.path,
-            resource: mapping.id,
-          },
-        }).unwrap();
-        history.push(
-          `/sources/${sourceId}/mappings/${mappingId}/attributes/${attribute.id}`
-        );
+        try {
+          const attribute = await createAttribute({
+            attributeRequest: {
+              definition_id: node.type,
+              path: node.path,
+              resource: mapping.id,
+            },
+          }).unwrap();
+          createInputGroup({ inputGroupRequest: { attribute: attribute.id } });
+          history.push(
+            `/sources/${sourceId}/mappings/${mappingId}/attributes/${attribute.id}`
+          );
+        } catch (error) {
+          console.error(error);
```

Comment:
Can you add a TODO comment here? Maybe we can try to work on a better error management in the future.